Bodekaer/Labster.EdX
# WEB-2864: Ensure more secure end user password policy

**Description:** Increase password complexity

**JIRA:** https://liv-it.atlassian.net/browse/WEB-2864

**Merge deadline:** List merge deadline (if any)

**Configuration instructions:** List any non-trivial configuration
instructions (if any).

**Testing instructions:**
https://edx-eucalyptus-sandbox3.labster.com/
1. Please create a new account of student
2. Check validation on password field
- Password required to be at least 8 chars length 
- It should contain both alpha and numeric chars

**Reviewers:**
- [ ] @avkoval 
- [ ] @TamoshaytisV 

**Merge checklist:**

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant
solutions, hacks, quick-and-dirty implementations, concerns about
migrations, etc.
